tasks/install_tasks.py

```diff
@@ -85,3 +85,11 @@ def install_shellcheck(ctx, version="0.8.0", destination="/usr/local/bin"):
     )
     ctx.run(f"cp \"/tmp/shellcheck-v{version}/shellcheck\" {destination}")
     ctx.run(f"rm -rf \"/tmp/shellcheck-v{version}\"")
+
+
+@task
+def install_devcontainer_cli(ctx):
+    """
+    Install the devcontainer CLI
+    """
+    ctx.run("npm install -g @devcontainers/cli")
```

Comment:
It assumes `npm` is installed do you know if it is installed by default with the Datadog laptop setup or should we maybe give info on how to install it if needed?

Reply:
Yep that's installed by default on our laptops